Orange/preprocess/preprocess.py

```diff
@@ -326,8 +326,7 @@ class Randomize(Preprocess):
     >>> randomizer = Randomize(Randomize.RandomizeClasses)
     >>> randomized_data = randomizer(data)
     """
-    Type = Enum("Randomize",
-                "RandomizeClasses, RandomizeAttributes, RandomizeMetas")
+    Type = 1, 2, 4
```

Comment:
This is now no longer an enum. Why not something like this?

    Type = Enum("Randomize", dict(RandomizeClasses=1, RandomizeAttributes=2, RandomizeMetas=4))